<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-UMRR3LS" name="NUGUAPP_002_연결.연결_07_NUGU_Tmap" tests="2" failures="0" timestamp="16 Jun 2020 03:01:37 GMT" time="28.240" errors="1">
  <testcase name="TC_연결_334" time="24.252" classname="NUGUAPP_002_연결.연결_07_NUGU_Tmap"/>
  <testcase name="TC_연결_336" time="3.988" classname="NUGUAPP_002_연결.연결_07_NUGU_Tmap">
    <error type="junit.framework.ComparisonFailure" message="expected:&lt;NUGU 서비스 연결을 위해 T map 인증을 해주...&gt; but was:&lt;연결이 완료되었습니다.
이제 T map에서 NUGU와 대화해보...&gt;">
      <![CDATA[junit.framework.ComparisonFailure: expected:<NUGU 서비스 연결을 위해 T map 인증을 해주...> but was:<연결이 완료되었습니다.
이제 T map에서 NUGU와 대화해보...>
	at junit.framework.Assert.assertEquals(Assert.java:81)
	at junit.framework.Assert.assertEquals(Assert.java:87)
	at NUGUAPP_002_연결.연결_07_NUGU_Tmap.TC_연결_336(연결_07_NUGU_Tmap.java:64)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:400)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </error>
  </testcase> <!-- TC_연결_336 -->
</testsuite> <!-- NUGUAPP_002_연결.연결_07_NUGU_Tmap -->
